recce/state.py

```diff
@@ -479,11 +536,11 @@ def _export_state_to_s3_bucket(self, metadata: dict = None) -> Union[str, None]:
 
         s3_client = boto3.client("s3")
         s3_bucket_name = self.cloud_options.get("host").replace("s3://", "")
-        s3_bucket_key = f"github/{self.pr_info.repository}/pulls/{self.pr_info.id}/{RECCE_STATE_COMPRESSED_FILE}"
+        s3_bucket_key = f"{self.catalog}/{self.pr_info.repository}/pulls/{self.pr_info.id}/{RECCE_STATE_COMPRESSED_FILE}"
```

Comment:
If it's for preview, it's not compressed.

Reply:
fixed!